src/constants/KeyCodes.js

```diff
@@ -1,6 +1,8 @@
 export const keyCodes = {
   ENTER: "Enter",
-  SPACE: " "
+  SPACE: " ",
+  DOWN_ARROW: 40,
```

Comment:
use these - as the cods are deprecated
 
  ARROW_DOWN: "ArrowDown",
  ARROW_UP: "ArrowUp"

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode